programs/address-lookup-table/src/instruction.rs

```diff
@@ -104,6 +125,40 @@ pub fn create_lookup_table(
     (instruction, lookup_table_address)
 }
 
+/// Constructs an instruction to create a lookup table account and returns
+/// the instruction and the table account's derived address. This version
+/// differs from the original create lookup table instruction by not requiring
+/// the authority key to sign the transaction. This makes it possible to create
+/// lookup tables on behalf of another authority such as a PDA.
```

Comment:
What's the rationale for using a separate instruction over just relaxing the `is_signer` check in the program itself?

As a separate idea, if you want to make this more specific for PDAs, you could require the derivation path for the authority in the instruction data.  That way you're proving that this is for a PDA, where generating a "signature" may not be possible without changes to a program.

Reply:
It might be better to just relax the `is_signer` check as you're suggesting.

My rationale is based on the fact that address lookup table creation is not idempotent and only one address lookup table can be created per slot for a given signer. By simply relaxing the authority signer check, this allows anyone to front run a transaction which creates a lookup table account and causes the front-ran transaction to fail.

In light of the above, I opted to introduce a new instruction which doesn't require changing either the idempotent behavior or the restriction of requiring the address in the derived path to be a signer. Another approach would be to make the instruction idempotent and relax the authority `is_signer` check. I think that's probably more intuitive, I'll explore that direction.

Reply:
I like the idea of relaxing the check and making the instruction idempotent.  That way, if someone front-runs your transaction, it doesn't make any difference at all.  The end result is exactly the same: you have an address lookup table, at the desired address, owned by you.